ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -197,6 +197,10 @@
   }
 }
 
+.ddp-required-question-prompt:after {
```

Comment:
Just FYI, CSS Selector Level 3 spec now differentiates between pseudo elements using single-colon (`:hover`, `:focus`) and pseudo classes using double-colon (`::before`, `::after`, `::placeholder`).